src/Propel/Generator/Command/MigrationDiffCommand.php

```diff
@@ -100,7 +100,7 @@ protected function execute(InputInterface $input, OutputInterface $output): int
         $manager->setWorkingDirectory($generatorConfig->getSection('paths')['migrationDir']);
 
         if ($manager->hasPendingMigrations()) {
-            throw new RuntimeException('Uncommitted migrations have been found ; you should either execute or delete them before rerunning the \'diff\' task');
+            throw new RuntimeException('Uncommitted migrations '.implode(', ', $manager->getValidMigrationTimestamps()).' have been found ; you should either execute or delete them before rerunning the \'diff\' task');
```

Comment:
You can use sprintf() to make it more readable:

sprtinf('Uncommitted migrations have been found (%s). You ...'

Reply:
Changed ;)